src/commands/generate/__tests__/generatorTest.ts

```diff
@@ -168,37 +157,29 @@ function testGenerateAllFilesAreTheSame(
 
         await generate(conjureDefinition, outputDir, typeGenerationFlags);
 
-        expectAllFilesAreTheSame(conjureDefinition, outputDir, actualTestCaseDir, typeGenerationFlags);
+        expectIdenticalDirectoryTrees(outputDir, actualTestCaseDir);
     };
 }
 
-function expectAllFilesAreTheSame(
```

Comment:
The old code just check if Conjure types matched up. The new code validates that the whole directory tree is identical.

This lets you catch bugs in the `index.ts` files, and also prevents you from checking in code that is not actually generated by any test (and which may diverge from the actual generator behavior).